src/client.js

```diff
@@ -160,7 +175,8 @@ export default class StitchClient {
         useRefreshToken: true,
         rootURL: this.rootURLsByAPIVersion[v2][API_TYPE_CLIENT]
       }
-    ).then(() => this.auth.clear()); // eslint-disable-line space-before-function-paren
+    ).then(() => this.auth.clear())
+      .then(() => true);
```

Comment:
q: why is this needed?

Reply:
As a successful return value. `clear()` doesn't produce anything useful– it would be nice to have a primitive value to check on the success of the logout.

Reply:
but the logout is implicitly successful if the promise that it returns doesn't reject, which the caller can check for with `.then`/`.catch`. It doesn't look like we use the `true` value anyway, so i don't see the point

Reply:
We use it in the new authenticate method.